clients/web/src/views/App.js

```diff
@@ -221,7 +223,8 @@ var App = View.extend({
 
             settings = _.extend(settings, {
                 el: this.$('#g-app-body-container'),
-                parentView: this
+                parentView: this,
+                brandName: this.brandName
```

Comment:
@brianhelba, what do you think of merging the PR without a full-on settings registry right now, then working on that next? @mgrauer may have an opinion on this as well.